dmscripts/helpers/env_helpers.py

```diff
@@ -1,9 +1,9 @@
 def get_api_endpoint_from_stage(stage, app='api'):
-    """Return the full URL of given API or Search API environment.
+    """Return the full URL of given API, Search API or web URL environment.
 
     :param stage: environment name. Can be one of 'preview', 'staging',
                   'production' or 'dev' (aliases: 'local', 'development').
-    :param app: should be either 'api' or 'search-api'
+    :param app: should be either 'api', 'search-api' or 'www'
```

Comment:
I did originally have a separate function for `www`, but it was nearly identical so I used this one to keep things DRYer. I'd be happy to rename the function to something more generic but I think you're right, we should be passing the domain as an arg to the script. It must be part of the environment context somewhere already, right?